lib/src/test/java/tech/relaycorp/awaladroid/endpoint/FirstPartyEndpointTest.kt

```diff
@@ -68,23 +68,24 @@ internal class FirstPartyEndpointTest : MockContextTestCase() {
         whenever(gatewayClient.registerEndpoint(any())).thenReturn(
             PrivateNodeRegistration(
                 PDACertPath.PRIVATE_ENDPOINT,
-                PDACertPath.PRIVATE_GW
+                PDACertPath.PRIVATE_GW,
+                ""
```

Comment:
Following up on the comment above, this shouldn't be an empty string, and we should even write a test that ensures this is stored and exposed to the app.